portfolio/src/main/webapp/process.php

```diff
@@ -0,0 +1,5 @@
+
+<?php
+   echo("First name: " . $_POST['firstname'] . "<br />\\n");
+   echo("Last name: " . $_POST['lastname'] . "<br />\\n");
+?>
```

Comment:
Is this file doing anything?  It's been a while since I've used PHP, but it kind of just looks like it's printing text.  Do you want this and email.html in this PR?  It seems like it's a partial feature that isn't really relevant to this PR, which seems to have the life map as its focus.